test/specs/profiles/visibility-agg.js

```diff
@@ -0,0 +1,165 @@
+var page = require('../../pages/profiles/view.js');
+var ManageDojoUsersPage = require('../../pages/dojos/manage-users');
+var MyDojosPage = require('../../pages/myDojos');
+var MainPage = require('../../pages/page');
+var profiles = require('../../data/profiles.js');
+var _ = require('lodash');
+
+describe('Test profile visibility', function () {
+  this.timeout(120000);
+
+  var fields = ['name', 'email', 'dojos']; // Fields not listed here doest not necessarly display
+  var tests = {
+    // 'manager1': profiles.full, // CDF admin
+    //
+    'champion1': profiles.full,
+    'mentor1': profiles.limited,
+    'mentorTicketing1': profiles.full,
+    'mentorAdmin1': profiles.full,
+    'mentorFullAccess1': profiles.full,
+    'parent1': profiles.limitedFamily,
+    'parentTicketing1': profiles.full,
+    'parentAdmin1': profiles.full,
+    'parentFullAccess1': profiles.full,
+    'child2': profiles.limitedFamily,
+    'randomParent1': profiles.none,
+
+    // Out of dojo
+    'champion2': profiles.otherDojo,
+    'parent2': profiles.otherDojo
+
+  };
+  var urls = {};
+
+  before(function () {
+    function getDojoUsers (n) {
+      return promiseSeries([
+        () => MainPage.open(),
+        () => MainPage.login('champion' + n + '@example.com', 'testchampion' + n),
```

Comment:
I'd rather getDojoUsers took in a user and dojo definition. Passing in just a number feels...wrong...